examples/template.html

```diff
@@ -124,6 +124,10 @@
             animate();
           });
 
+          $(document).ready(function(){
```

Comment:
Not possible.

On Tue, Jan 26, 2016 at 9:41 AM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> In examples/template.html
> <https://github.com/biocore/emperor/pull/442#discussion_r50870625>:
>
> > @@ -124,6 +124,10 @@
> >              animate();
> >            });
> >
> > +          $(document).ready(function(){
>
> Can we just remove this? Or is that not possible?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/emperor/pull/442/files#r50870625>.
>